src/lib/expand-unit.ts

```diff
@@ -100,6 +102,40 @@ export const expandCrossbellNote = async ({
       ...new Set(expandedNote.metadata.content.images),
     ]
 
+    if (useScore) {
```

Comment:
Repeat with the following code below. https://github.com/Crossbell-Box/xLog/pull/1059/files#diff-0802612f2ebfab23d2656fc2189271c4fef6451452a2bd7dacc8d8bcfe11e2e8R193